x-pack/plugins/security_solution/public/common/mock/endpoint/react_dom.mocks.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Mocks the React DOM module to ensure compatibility with react-testing-library and avoid
+ * error like:
+ *
+ * ```
+ * TypeError: parentInstance.children.indexOf is not a function
+ *       at appendChild (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:7183:39)
+ * ```
+ *
+ * For more information on this need, see:
+ *
+ * @see https://github.com/facebook/react/issues/11565
+ *
+ * @example
+ *
+ * jest.mock('react-dom', () => {
+ *   return jest
+ *     .requireActual('../path/to/this/react_dom.mocks')
+ *     .getReactDomMock();
+ * });
+ */
+export const getReactDomMock = () => {
+  // FIXME:PT delete this utility - not used?
+  const realReactDomModule = jest.requireActual('react-dom');
+
+  return {
+    ...realReactDomModule,
+
+    createPortal: jest.fn((...args) => {
+      realReactDomModule.createPortal(...args);
+      // Needed for react-Test-library. See:
+      // https://github.com/facebook/react/issues/11565
+      return args[0];
+    }),
+  };
+};
```

Comment:
This is not being used anywhere. No?

Reply:
Correct. Not right now. I thought I was going to need it for other security solution tests, since the Responder overlay was present in DOM at all times, but I have changed that as well so that it's only present withe displayed. I will remove this module